detect_secrets/core/usage.py

```diff
@@ -145,18 +146,19 @@ def _add_output_verified_false_flag(self):
         add_output_verified_false_flag(self.parser)
         return self
 
-    def _add_fail_on_unaudited_flag(self):
+    def _add_fail_on_non_audited_flag(self):
         self.parser.add_argument(
-            '--fail-on-unaudited',
+            '--fail-on-non-audited',
```

Comment:
Can this change please be undone? This renames the `--fail-on-unaudited` flag and will break detect-secrets for users who are using this flag in their builds and pre-commit config files.

Reply:
I think I understand what happened, you're probably working off an outdated version of master (see https://github.com/IBM/detect-secrets/pull/118#discussion_r1175851211).